### DISCUSSION TITLE
do-release-upgrade for Ubuntu 18.04 to 20.04

### DISCUSSION THREAD

**Discussion by kwwv** [Security-Onion-Solutions/securityonion#9713] [Unsupported Versions]
We have three search nodes and a manager node running Ubuntu 18.04. We need to migrate to 20.04 as 18.04 is EOL soon. 

Does anyone have experience (and/or tips) on running do-release-upgrade to upgrade the system in place? Are there any gotchas to be aware of?

**Comment by cm-ops** [comment]
This will work https://github.com/Security-Onion-Solutions/securityonion/discussions/8546 I tested as well, and have an almost similar process.

**Reply by petiepooo** [reply]
SOS devs, can we please get an officially supported procedure for this upgrade?  Ubuntu 18.04 will go into extended maintenance in about two months.

**Comment by cm-ops** [comment]
https://github.com/Security-Onion-Solutions/securityonion/discussions/9776
https://docs.securityonion.net/en/latest/appendix-b.html
